recipes/duckdb-extensions-split/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "duckdb-extension-json" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/duckdb/duckdb/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 662a0ba5c35d678ab6870db8f65ffa1c72e6096ad525a35b41b275139684cea6
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - cmake
+
+test:
+  commands:
+      - find "${PREFIX}"/duckdb/extensions/v{{ version }}/ -type f -name 'json.duckdb_extension' | grep '.'
+      - if [ "$(duckdb -json -c "set extension_directory='${PREFIX}/duckdb/extensions'; select extension_name from duckdb_extensions() where (installed and install_path != '(BUILT-IN)');" | jq -r '.[].extension_name')" = "json" ]; then exit 0; else exit 1; fi
```

Comment:
Note: DuckDB searches for extensions in the user's home directory. This behavior is currently not configurable from the outside, only inside a DuckDB session. See [here](https://github.com/duckdb/duckdb/discussions/9845#discussioncomment-9277190) for details.